onnxruntime/core/graph/graph_utils.cc

```diff
@@ -228,8 +252,8 @@ static void MoveAllNodeOutputs(Graph& graph, Node& src_node, Node& target_node)
   auto output_edges = GetNodeOutputEdges(src_node);
 
   for (auto cur = output_edges.cbegin(), end = output_edges.cend(); cur != end; ++cur) {
-    graph.AddEdge(target_idx, cur->dst_node, 0, cur->dst_arg_index);
-    graph.RemoveEdge(src_idx, cur->dst_node, 0, cur->dst_arg_index);
+    graph.AddEdge(target_idx, cur->dst_node, cur->src_arg_index, cur->dst_arg_index);
+    graph.RemoveEdge(src_idx, cur->dst_node, cur->src_arg_index, cur->dst_arg_index);
```

Comment:
Same here -- use RemoveGraphEdges(graph, output_edges) just to not worry about getting the arguments right.